src/client/NetworkingClient.cpp

```diff
@@ -86,6 +91,34 @@ void NetworkingClient::RegisterNetworkingClient() {
     }
 }
 
+void NetworkingClient::ApiClientRecv() {
+    while (true) {
+        sf::Packet group_response;
+        ApiCommand api_command;
+
+        if (mApiClient->receive(group_response) == sf::Socket::Done) {
+            if (group_response >> api_command &&
+                api_command.command == (sf::Uint32)APICommandType::group) {
```

Comment:
Calling this Command `group` is confusing and vague

Reply:
alternative? maybe it should be toggle_group?

Reply:
`toggle_groupability` sgtm

Reply:
fixed in 92f3e38